src/odemis/driver/tfsbc.py

```diff
@@ -160,12 +160,14 @@ class BeamShiftController(model.HwComponent):
     through the MD_CALIB metadata (a 4x2 tuple, 4x (float, float), xlower, xupper, ylower, yupper).
     """
 
-    def __init__(self, name, role, port=None, serialnum=None, **kwargs):
+    def __init__(self, name, role, port=None, serialnum=None, dependencies=None, **kwargs):
         """
         :param port (str): (e.g. "/dev/ttyUSB0") or pattern for port ("/dev/ttyUSB*"),
             "/dev/fake" will start the simulator
         :param serialnum (str): serial number of RS485 adapter
-        The connection can be specified by either port or serialnum, it's not needed to provide both.
+            The connection can be specified by either port or serialnum, it's not needed to provide both.
+        :param dependencies (dict str -> scanner):
+            scanner component -> name of the xt multibeam scanner component.
```

Comment:
Because it's optional to provide the `dependencies`, can you update this with a short description of what happens when you don't provide the scanner dependency (`dependencies=None`)